### DISCUSSION TITLE
Wrong label for an image in the Food-101 dataset

### DISCUSSION THREAD

**Discussion by exenGT** [mrdbourke/pytorch-deep-learning#685] [General]
It's interesting to find out that the image pizza_steak_sushi/test/pizza/2508636.jpg shows a hamburger and some fries instead of a pizza:

![2508636](https://github.com/mrdbourke/pytorch-deep-learning/assets/13700125/50b7b59d-518a-472a-b1f3-21f6e60716f3)

Which makes me wonder if there are any other wrongly labeled images out there in this dataset...

**Comment by mrdbourke** [comment]
Hey @exenGT , 

You're right, there are a few errors in the original dataset (Food-101) as well as the subsets I've created (I didn't change any original labels).

This is a common issue for larger scale datasets.

It's hard to make sure every single label in a dataset is correct.

For example, ImageNet may have 1,000s of incorrect labels, see here: https://labelerrors.com/ 

There are tools like `cleanlab` which help to find these errors and offer ways to explore/fix them: https://github.com/cleanlab/cleanlab
